api/send-email.js

```
@@ -33,23 +33,46 @@ export default async function handler(req, res) {
 
     switch (type) {
       case 'chatbot_summary':
+        // Validate conversation data exists
+        const hasConversation = conversationData?.full_conversation && conversationData.full_conversation.trim().length > 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)